utils/threshAlgos.py

```diff
@@ -1229,19 +1251,42 @@ def sbitRateAnalysis(chamber_config, rateTree, cutOffRate=0.0, debug=False, outf
                     pass
                 pass
             pass
+
+        # Make Graphs /////////////////////////////////////////////////////////////////////////////
+        #++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
         for idx in range(len(dacNameArray)):
             dacName = np.asscalar(dacNameArray[idx])
             if perchannel:
                 canv_Summary2D = make3x8Canvas("canv_Summary_Rate2D_vs_{0}".format(dacName),dict_vfatCHVsDACNameX_Rate2D[dacName][ohKey],"TRI1")
                 for vfat in range(1,25):
                     canv_Summary2D.cd(vfat).SetLogz()
             else:
-                canv_Summary1D = make3x8Canvas("canv_Summary_Rate1D_vs_{0}".format(dacName),dict_Rate1DVsDACNameX[dacName][ohKey],"APE1")
-                for vfat in range(1,25):
-                    canv_Summary1D.cd(vfat).SetLogy()
-                    dict_Rate1DVsDACNameX[dacName][ohKey][vfat].GetYaxis().SetRangeUser(1e-1,1e8)
-                    canv_Summary1D.cd(vfat).Update()
-
+                canv_Summary1D = make3x8Canvas("canv_Summary_Rate1D_vs_{0}".format(dacName),dict_Rate1DVsDACNameX[dacName][ohKey],"APE1" )
+                # make 24 TLines
+                kneeLine= []
+                for vfat in range(0,24):
+                    canv_Summary1D.cd(vfat + 1).SetLogy()
+
+                    # make TH1F into TGraph
+                    graph = dict_Rate1DVsDACNameX[dacName][ohKey][vfat]
+                    if type(graph) == r.TH1F :
+                        graph = r.TGraph(graph)
+
+                    # get maximum y value
+                    y = graph.GetY()
+                    y = np.array(y)
+                    ymax = np.amax(y)
+
+                    # Draw a line on the graphs
+                    kneeLine.append(r.TLine(dict_dacInflectPts[dacName][ohKey][vfat][0], 10.0, dict_dacInflectPts[dacName][ohKey][vfat][0], ymax) )
+                    kneeLine[vfat].SetLineColor(2)
+                    kneeLine[vfat].SetVertical()
+                    canv_Summary1D.cd(chamber_vfatPos2PadIdx[vfat] )
+                    kneeLine[vfat].Draw()
```

Comment:
is this line position provided to the user in any tabular format or output data that could be read that would *not* require opening the TFile?

Reply:
I do not quite follow what you are looking for here.

Reply:
How does someone running this analysis extract from the output the value of the `ARM_DAC` that the inflection point is at?
Can you print a table that says something like

| `VFAT` | `ARM_DAC` inflection point |
|----|----|
| 0 | 32 |
| 1 | 54 |
| 2 | 12 |

Reply:
Do you want me to put this in a text file? Or just have it print these values as the code runs?

Reply:
Update from @bdorney:

Do both. As the code run prints the following:
| OHN | VFATN | ARM DAC Inflection Pt |
|--------|-----------|-------|
|   0     |     0       | 20  |
|   3     |    17	  | 45  |

	
and the output text file needs to be sent to the appropriate location under $DATA_PATH e.g. $DATA_PATH/detectorName/....etc...

For the text file drop the OHN column, since it will be only for one link.

Reply:
Okay, I added it and pushed